fastapi_contrib/db/models.py

```diff
@@ -66,6 +66,12 @@ def get_db_collection(cls):
         db = get_db_client()
         insert_result = await db.insert(self, include=include, exclude=exclude)
         self.id = insert_result.inserted_id
+    
+    @async_timing
+    async def update(self, include=None, exclude=None):
+        db = get_db_client()
+        insert_result = await db.update(self, include=include, exclude=exclude)
```

Comment:
We could use better naming here.

On the second thought, do we even need to assign `id` once more? I think it is already assigned at the serializer -> model conversion time.